independent-publisher-connectors/Mapbox/apiDefinition.swagger.json

```diff
@@ -0,0 +1,892 @@
+{
```

Comment:
@SimonL26 the contact section is missing, please refer to [apiDefinition template](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/Independent%20Publisher/apiDefinition.swagger.json).